package.json

```diff
@@ -60,21 +60,21 @@
     "test:lint": "eslint katex.js katex.webpack.js cli.js webpack.common.js webpack.config.js webpack.dev.js src static test contrib dockers && stylelint src/katex.less",
     "test:flow": "flow",
     "test:jest": "jest",
-    "jest-update": "jest --updateSnapshot",
-    "coverage": "jest --coverage",
+    "test:jest:update": "jest --updateSnapshot",
+    "test:jest:coverage": "jest --coverage",
+    "test:screenshots": "npm run test:screenshots:update -- --verify",
+    "test:screenshots:update": "npm run prestart && dockers/Screenshotter/screenshotter.sh",
+    "test:perf": "npm run prestart && NODE_ENV=test node test/perf-test.js",
     "clean": "rm -rf build/* node_modules/",
     "clean-install": "npm run clean && npm i",
```

Comment:
These were added in #902, but I'm not sure there is a use case. Also, I think it may be confused with only removing `build` folder.